app/models/course.rb

```diff
@@ -1,6 +1,11 @@
 class Course < ActiveRecord::Base
   validates :title, length: { maximum: 50 }, presence:  true
   validates :description, presence:  true
+  belongs_to :user, dependent: :destroy
 
   mount_uploader :image, CourseImageUploader
+
+  def belongs_to?(user)
```

Comment:
rename it. We have defined belongs_to method for relations in rails. It can cause confusion.